py4DSTEM/process/utils/utils.py

```diff
@@ -260,31 +260,6 @@ def add_to_2D_array_from_floats(ar, x, y, I):
     ar[x1[mask], y1[mask]] += (    dx[mask]) * (    dy[mask]) * I[mask]
     return ar
 
-
-def print_progress_bar(iteration, total, prefix='', suffix='', decimals=1,
```

Comment:
Has this func been removed/replaced with Tqdmnd?

Reply:
yes